src/components/explorer/IndividualVariants.js

```diff
@@ -26,28 +20,13 @@ const variantExpressionPropType = PropTypes.shape({
     version: PropTypes.string,
 });
 
-const VariantExpressionDetails = ({variantExpression, geneContext}) => {
-    const dispatch = useDispatch();
-    const { individualID } = useContext(ExplorerIndividualContext);
-    const tracksUrl = useMemo(() => {
-        if (individualID) {
-            return `${explorerIndividualUrl(individualID)}/tracks`;
-        }
-    }, [individualID]);
+const VariantExpressionDetails = ({variantExpression}) => {
```

Comment:
Can be wrapped in a `memo`

Reply:
Also modify the rendering code to remove the `gene_context` prop (line 64) and conditional render (line 61)